src/common/Smi.Common/Options/GlobalOptions.cs

```diff
@@ -486,7 +486,15 @@ public class DicomAnonymiserOptions : IOptions
         public string? RoutingKeySuccess { get; set; }
         public string? RoutingKeyFailure { get; set; }
         public bool FailIfSourceWriteable { get; set; } = true;
-
+        public string? VirtualEnvPath { get; set; }
```

Comment:
We can simplify this by passing the Python interpreter path and also avoid the need to activate the virtual environment. Maybe also rename it to `PythonInterpreter`?

Reply:
Discussion: (da, rkm) dicom_pixel_anon.sh requires the virtual environment to be activated.
Action: (da, rkm) Review the script for this change once the core requirements are completed.